Core/Aspects/Caching/CacheAspect.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Castle.DynamicProxy;
+using WebNetSample.Core.CrossCuttingConcerns.Caching;
+using WebNetSample.Core.Utilities.Interceptors;
+using WebNetSample.Core.Utilities.IoC;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace WebNetSample.Core.Aspects.Caching;
+
+public class CacheAspect : MethodInterception
+{
+    private readonly ICacheService _cacheService;
+    public readonly int duration;
+    public string Location { get; set; } = "Client";
+
+    public CacheAspect(int duration = 60)
```

Comment:
Making a default value in constructor is definitely a bad idea. I see an option when you are adding overload for constructor with `int` param and moving constant with default value into private class constant to assign it in default constructor, like:
```suggestion
    public CacheAspect()
    {
         _duration = DefaultDuration;
    }

    public CacheAspect(int duration)
    {
         _duration = duration
    }
```